spec/feature/peeps_options_spec.rb

```diff
@@ -0,0 +1,21 @@
+feature 'peeps' do
+  
+  scenario "it should show a list of peeps" do
+    Peep.create(title: "Test", peep: "this is a test peep")
+    Peep.create(title: "Hello", peep: "feeling ok")
+    Peep.create(title: "Bye", peep: "feeling sad")
```

Comment:
I would not create things directly in code within a feature test. The whole test should be from the point of view of the user!